baggage/HTTP_HEADER_FORMAT.md

```diff
@@ -1,16 +1,16 @@
-# Correlation Context HTTP Header Format
+# Baggage HTTP Header Format
 
-The correlation context header is used to propagate user-supplied key-value pairs through a trace.
+The baggage header is used to propagate user-supplied key-value pairs through a trace.
```

Comment:
Text keeps referring to "trace", which is only tangentially related. It would be useful to introduce a term of "distributed request" or "distributed execution" at the beginning and use that, instead of "trace".

Reply:
I agree with this point. The person reading the spec should not have to be familiar with the term `trace` or the trace context spec.

Reply:
Let's look through the rest of the wording after the renaming is done. Issue created.